src/index.d.ts

```diff
@@ -0,0 +1,180 @@
+// Type definitions for @socialgouv/kali-data
+
+/**
+ * Get a full agreement unist tree with its sections and articles.
+ *
+ * @see https://github.com/syntax-tree/unist
+ */
+export function getAgreement(agreementIdOrIdcc: number | string): KaliData.Agreement;
+
+/**
+ * Get a flat unist array of all the articles an agreement contains.
+ *
+ * @see https://github.com/syntax-tree/unist
+ */
+export function getAgreementArticles(
+  agreementIdOrIdcc: number | string,
+): KaliData.AgreementArticleWithSections[];
+
+/**
+ * Get an agreement article unist node.
+ *
+ * @see https://github.com/syntax-tree/unist
+ */
+export function getArticle(articleId: string): KaliData.AgreementArticleWithSections;
+
+/**
+ * Get the full list of indexed agreements.
+ */
+export function getAgreements(): KaliData.IndexedAgreement[];
+
+/**
+ * Get the full list of indexed articles.
+ */
+export function getArticles(): KaliData.IndexedArticle[];
+
+/**
+ * Check if an agreement is available.
+ */
+export function hasAgreement(agreementIdOrIdcc: number | string): boolean;
+
+/**
+ * Check if an article is available.
+ */
+export function hasArticle(articleId: string): boolean;
+
+export as namespace KaliData;
+
+/**
+ * TODO Check and describe types.
+ */
+
+/**
+ * Agreement or Article State.
+ *
+ * - ABROGE: ...
+ * - DENONCE: ...
+ * - MODIFIE: ...
+ * - PERIME: ...
+ * - REMPLACE: ...
+ * - VIGUEUR: ...
+ * - VIGUEUR_ETEN: ...
+ * - VIGUEUR_NON_ETEN: ...
+ */
+type State =
+  | "ABROGE"
+  | "DENONCE"
+  | "MODIFIE"
+  | "PERIME"
+  | "REMPLACE"
+  | "VIGUEUR"
+  | "VIGUEUR_ETEN"
+  | "VIGUEUR_NON_ETEN";
+
+type Agreement = {
+  type: "section";
+  data: AgreementData;
+  children: AgreementSection[];
```

Comment:
I think parent is missing

Reply:
Why? I haven't seen any `parent` key in the unist tree... It need to be mapped via `unist-util-parents` in order to get a `parent` pointer for each node.

Reply:
Oh I see, you use parent but you remove it https://github.com/SocialGouv/kali-data/pull/96/files/a1af4476bb68dac98e63df63ea3632c844f1cd7d#diff-347b07d9598ee2be7edc2db0e84d9b3aR37